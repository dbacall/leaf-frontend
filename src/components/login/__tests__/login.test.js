import React from 'react';
import { shallow } from 'enzyme';
import Login from '../loginComponent';

describe('Login component tests:', () => {
  const props = {
    errors: {},
  };

  it('should render correctly', () => {
    const wrap = shallow(<Login {...props} />);
    expect(wrap).toMatchSnapshot();
  });
});
