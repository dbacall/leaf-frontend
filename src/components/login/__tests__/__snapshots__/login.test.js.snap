// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component tests: should render correctly 1`] = `
<section
  className="login"
>
  <div
    className="formContainer"
  >
    <div>
      <h3>
        Login
      </h3>
      <p>
        Don't have an account? 
        <Link
          to="/register"
        >
          Register
        </Link>
      </p>
    </div>
    <form
      noValidate={true}
      onSubmit={[Function]}
    >
      <input
        className=""
        id="email"
        onChange={[Function]}
        placeholder="Email"
        type="email"
        value=""
      />
      <p
        className="warning"
      />
      <input
        className=""
        id="password"
        onChange={[Function]}
        placeholder="Password"
        type="password"
        value=""
      />
      <p
        className="warning"
      />
      <button
        className="submitBtn"
        type="submit"
      >
        Login
      </button>
    </form>
  </div>
</section>
`;
